maintenance/ebpf/ebpf-and-eks.md

```diff
@@ -0,0 +1,193 @@
+  your VPC subnet and has the `natOutgoing` flag set.  The IP pool will now be used for IPAM since AWS VPC CNI has its own
+  IPAM, but it will tell {{site.prodname}} to SNAT traffic that is leaving the confines of your VPC.
+  
+  ```
+  calicoctl apply -f - <<EOF 
+  apiVersion: projectcalico.org/v3
+  kind: IPPool
+  metadata:
+    name: vpc-subnet
+  spec:
+    cidr: <your VPC subnet>
+    natOutgoing: true
+    nodeSelector: !all()
+  EOF
+  ```  
+ 
+%>
+<label:Custom AMI>
+<%
+
+
+#### Option 2: Create a custom AMI
+
+If you are familiar with the AMI creation process, it is also possible to create a custom AMI based on Ubuntu 20.04, 
+which is suitable:
+
+* Create an instance from the default EKS Ubuntu image.
+
+* Log into the instance with `ssh` and upgrade it to Ubuntu 20.04.
+
+* [Save the instance off as a custom AMI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/creating-an-ami-ebs.html){:target="_blank"} 
+  and make a note of the AMI ID
+
+* Using `eksctl`: start your cluster as normal, but when creating the nodegroup, add the `--node-ami` and
+  `--node-ami-family` settings.
+
+  * `--node-ami` should be set to the AMI ID of the image built above.
+  * `--node-ami-family` should be set to `Ubuntu1804` (in spite of the upgrade).
+
+  For example:
+  ```
+  eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --node-ami auto --max-pods-per-node 100 --node-ami-family Ubuntu1804 --node-ami <AMI ID>
+  ```
+ 
+* To use {{site.prodname}} with the AWS VPC CNI: 
+
+  * install {{site.prodname}} using the following pre-release manifest from the AWS VPC CNI project:
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/56851f0905dba4852eb895ec1c7bd5b1876a9c67/config/master/calico.yaml
+    ```
+  
+    > **Note**: It's important to use this pre-release manifest because the released version uses a version of {{site.prodname}}
+    > that is too old and only has partial support for eBPF mode.
+    {: .alert .alert-info}
+
+  * [Install `calicoctl`]({{site.baseurl}}/getting-started/clis/calicoctl/install); it is needed for the following step.
+
+  * To work around an incompatibility between the AWS VPC CNI and eBPF mode, create a {{site.prodname}} IP pool that matches
+    your VPC subnet and has the `natOutgoing` flag set.  The IP pool will now be used for IPAM since AWS VPC CNI has its own
+    IPAM, but it will tell {{site.prodname}} to SNAT traffic that is leaving the confines of your VPC.
+  
+    ```
+    calicoctl apply -f - <<EOF 
+    apiVersion: projectcalico.org/v3
+    kind: IPPool
+    metadata:
+      name: vpc-subnet
+    spec:
+      cidr: <your VPC subnet>
+      natOutgoing: true
+      nodeSelector: !all()
+    EOF
+    ```  
+
+* Alternatively, follow the "Install EKS with {{site.prodname}} networking" section of the 
+  [this guide](../../getting-started/kubernetes/managed-public-cloud/eks.md).
```

Comment:
This link is 404 for me

Reply:
Hmm.  Looking at the instructions in that link, it covers the `eksctl create cluster` step.  Is it safe to assume people will understand that they've already created the eks cluster?  

In the end, you're only really wanting 2 commands from that procedure: 
`kubectl delete daemonset -n kube-system aws-node` 
and 
`kubectl apply -f https://deploy-preview-4074--calico-master.netlify.app/manifests/calico-vxlan.yaml`
perhaps just add those commands in here instead of jumping to the other doc?

Reply:
Ah - also - because you didn't create the nodepool with 0 nodes in it before scaling up the cluster, you also need to restart the coreDNS pods (they'll already be up and networked by aws-cni)